model/src/pyrenew/process/ar.py

```diff
@@ -91,7 +91,7 @@ def _ar_scanner(carry, next):  # numpydoc ignore=GL08
         )
 
         last, ts = lax.scan(_ar_scanner, inits - self.mean, noise)
-        return (jnp.hstack([inits, self.mean + ts.flatten()]),)
+        return (SampledValue(jnp.hstack([inits, self.mean + ts.flatten()])),)
```

Comment:
As above, I think we should probably pass `self.t_start` and `self.t_unit` here, even if they're typically going to be `None`.

Reply:
Noted